signbank/dictionary/migrations/0017_auto_20220601_1151.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.11 on 2022-05-31 23:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dictionary', '0016_auto_20220601_1059'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='gloss',
+            name='fingerspelling',
+            field=models.BooleanField(blank=True, default=False, null=True),
```

Comment:
Can we make this `blank=False, null=False`? The field should never be empty, and shoudn't be allowed to be empty - it should either be `True`or `False`